data/hails.txt

```diff
@@ -436,11 +451,13 @@ phrase "friendly civilian"
 		"See you later"
 		"See you round the 'verse"
 		"Stay safe"
+		"Safe travels"
+		"Watch out for those pirates
 	word
 		", "
 	word
-		"fellow captain"
-		"captain"
+		"fellow Captain"
```

Comment:
Captain isn't being used as a proper noun in this case and therefore doesn't need capitalized,
"fellow captain" vs "Hello, Captain". In the first one, captain refers to any ship owner, while in the second it refers to the specific captain in question.

The best parallel I can make is how "mom" or "dad" is only capitalized if you're actually talking to someone and calling them "mom" or "dad."

Reply:
Thank you! I'll change that back to what it was then. Seems like it's being used as a title such as "President" it'd be capitalized.

Reply:
Titles aren't capitalized unless referring to a specific person with that title. "The President of the United States" would be capitalized since it refers to one specific person, but "presidents of the United States" would not be capitalized since it refers to multiple people of the same title.

"fellow captain" is an interesting situation, as while it is being addressed to a single person, it refers to the title of captain in a general sense.